config/deployer.sample.json

```diff
@@ -47,7 +47,8 @@
     "Password": "mostest80098bigpass_",
     "EnablePerformanceInsights": true,
     "DBParameters": [],
-    "ClusterIdentifier": ""
+    "ClusterIdentifier": "",
+    "EngineVersion": ""
```

Comment:
Let's have a default value here.

Reply:
Leaving it empy would pick the default value from the terraform variable:

```tf
variable "db_default_engine_version" {
  type = map(any)
  default = {
    "aurora-mysql"      = "8.0.mysql_aurora.3.05.2"
    "aurora-postgresql" = "14.7"
  }
}
```

Reply:
Low urgency here, but I think putting the default into the sample is helpful for the operator.